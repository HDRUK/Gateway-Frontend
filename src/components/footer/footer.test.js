import React from "react";
import { create } from "react-test-renderer";
import Footer from "./footer.js";
import hdr_logo_white from "../../assets/hdruk_logo_white.png";

describe("<Footer> ", () => {
    let renderedComponent;

    beforeEach(() => {
        renderedComponent = create(<Footer image={hdr_logo_white} />);
    });

    it("should test the Footer Snapshot", () => {
        expect(renderedComponent).toMatchSnapshot();
    });
});
