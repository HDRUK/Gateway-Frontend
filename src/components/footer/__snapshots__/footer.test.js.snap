// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer>  should test the Footer Snapshot 1`] = `
<div
  className="sc-hSdWYo cuoCdb"
>
  <div
    className="sc-eHgmQL cHykFZ"
  >
    <img
      className="sc-kGXeez sc-jWBwVP jtuAgN"
      src="hdruk_logo_white.png"
    />
  </div>
  <div
    className="sc-eHgmQL cHykFZ"
  >
    <div
      className="sc-cvbbAY eWqCsX"
    >
      Follow us on social media
    </div>
    <div
      className="sc-kpOJdX jePBvI"
    >
      <a
        href="https://twitter.com/HDR_UK"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          aria-hidden={true}
          focusable="false"
          height={32}
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "willChange": "transform",
            }
          }
          viewBox="0 0 32 32"
          width={32}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.92 24.94A12.76 12.76 0 0 0 24.76 12.1v-.59A9.4 9.4 0 0 0 27 9.18a9.31 9.31 0 0 1-2.59.71 4.56 4.56 0 0 0 2-2.5 8.89 8.89 0 0 1-2.86 1.1 4.52 4.52 0 0 0-7.7 4.11 12.79 12.79 0 0 1-9.3-4.71 4.51 4.51 0 0 0 1.4 6 4.47 4.47 0 0 1-2-.56v.05a4.53 4.53 0 0 0 3.55 4.45 4.53 4.53 0 0 1-2 .08A4.51 4.51 0 0 0 11.68 21a9.05 9.05 0 0 1-5.61 2A9.77 9.77 0 0 1 5 22.91a12.77 12.77 0 0 0 6.92 2"
          />
        </svg>
      </a>
    </div>
    <div
      className="sc-kpOJdX jePBvI"
    >
      <a
        href="https://www.linkedin.com/company/healthdataresearchuk/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <svg
          aria-hidden={true}
          focusable="false"
          height={32}
          preserveAspectRatio="xMidYMid meet"
          style={
            Object {
              "willChange": "transform",
            }
          }
          viewBox="0 0 32 32"
          width={32}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M26.21 4H5.79A1.78 1.78 0 0 0 4 5.73V26.2a1.77 1.77 0 0 0 1.79 1.73h20.42A1.77 1.77 0 0 0 28 26.2V5.73A1.78 1.78 0 0 0 26.21 4zm-15.1 20.41H7.59V13h3.52zm-1.72-13a2.07 2.07 0 0 1-2.07-2.02 2 2 0 0 1 2.07-2.07 2.07 2.07 0 0 1 0 4.13zm15.09 12.93H21v-5.58c0-1.33 0-3.06-1.86-3.06S17 17.16 17 18.63v5.65h-3.56V13h3.32v1.5h.07a3.72 3.72 0 0 1 3.39-1.86c3.59 0 4.26 2.4 4.26 5.45z"
          />
        </svg>
      </a>
    </div>
  </div>
  <div
    className="sc-eHgmQL cHykFZ"
  >
    <div
      className="sc-cvbbAY eWqCsX"
    >
      Visit the HDR UK Site
    </div>
    <div
      className="sc-cvbbAY eWqCsX"
    >
      Contact us
    </div>
    <div
      className="sc-cvbbAY eWqCsX"
    >
      Accessibility statement
    </div>
    <div
      className="sc-cvbbAY eWqCsX"
    >
      Terms and conditions
    </div>
  </div>
  <text
    className="sc-jTzLTM iPfJDU"
  >
    Â©HDR UK 2019. All rights reserved. | Design and development by IBM IX
  </text>
</div>
`;
